### DISCUSSION TITLE
Add Skript programming language (.sk files)

### DISCUSSION THREAD

**Discussion by daneedev** [community/community#108933] [Repositories]
### Select Topic Area

Question

### Body

Hey,
there is a scripting language for Minecraft, it's called Skript. I would like to see it in languages section in repository like this: ![](https://i.daneeskripter.dev/NdjGYu.png)

Here is the official repository: https://github.com/SkriptLang/Skript

**Comment by gil-son** [comment]
In this [link](https://docs.github.com/en/get-started/learning-about-github/github-language-support) you can check the supported programmings language on Github

Well, you can open a Ticket with Github Support


Acces the [Github Support](https://support.github.com/)

Click on My Ticket :

![github04](https://github.com/community/community/assets/72712095/096b3477-cee6-4a19-9f1b-6fb2219b51a1)

and open a New Ticket:

![github05](https://github.com/community/community/assets/72712095/7413fc55-875a-498c-93eb-5c54d5ffc8d6)


If my support has been helpful, please consider marking my answer. God bless you!

**Comment by daneedev** [comment]
> In this [link](https://docs.github.com/en/get-started/learning-about-github/github-language-support) you can check the supported programmings language on Github
> 
> Well, you can open a Ticket with Github Support
> 
> Acces the [Github Support](https://support.github.com/)
> 
> Click on My Ticket :
> 
> ![github04](https://private-user-images.githubusercontent.com/72712095/302150762-096b3477-cee6-4a19-9f1b-6fb2219b51a1.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MDg2ODExNzMsIm5iZiI6MTcwODY4MDg3MywicGF0aCI6Ii83MjcxMjA5NS8zMDIxNTA3NjItMDk2YjM0NzctY2VlNi00YTE5LTlmMWItNmZiMjIxOWI1MWExLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDAyMjMlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwMjIzVDA5MzQzM1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPThjMTc5NTExYTIxMzM3NmM0N2I3NDI1YTc2Yjg2OGUxMDViYWE2YjUzYzBmOTllNjYxNTMyZTc1YTViMWZiZWQmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.LKMz4QEKmf2Q-9BoJYcaLO2JgfKeEf1s3y2XEL2js-E)
> 
> and open a New Ticket:
> 
> ![github05](https://private-user-images.githubusercontent.com/72712095/302150897-7413fc55-875a-498c-93eb-5c54d5ffc8d6.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MDg2ODExNzMsIm5iZiI6MTcwODY4MDg3MywicGF0aCI6Ii83MjcxMjA5NS8zMDIxNTA4OTctNzQxM2ZjNTUtODc1YS00OThjLTkzZWItNWM1NGQ1ZmZjOGQ2LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDAyMjMlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwMjIzVDA5MzQzM1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTk3MTliMmUzMjU0MTVlYjdiMzU4YTFiNmFhZDY0NGJkODRhNjFjMjUyODllMDZiZGE4M2YzNWJiNDY1MTBiOWEmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.imIbn84QbvhsPE4SKHhgNW53Mw6G5PVnqvb-_tjOoO4)
> 
> If my support has been helpful, please consider marking my answer. God bless you!

Hey, thanks for answer, but I don't know which category for the ticket should I use.

**Comment by daneedev** [comment]
I sent them an email, so I will close this discusssion.
